Adding support for for HC pack for x8 (#36709)
demisto/content: Packs/HealthCheck/ReleaseNotes/3_0_0.md

#### Playbooks

##### HealthCheck

> Updated the main playbook to differentiate between Cortex XSOAR versions 6.x and 8.x.

Reviewer: Hi @tsachzim 
Usually, if we have the same playbook that works differently for v6 and v8, we will just create 2 versions of the playbook and will associate each one to the relevant platform version using the "marketplaces" key in the playbook file, for example, for the playbook which we want to run only on v6:
_**marketplaces: ["xsoar_on_prem"]**_

And for v8:
_**marketplaces: ["xsoar_saas"]**_

This makes the playbook more readable and I strongly advise to do so.
Since this is a community pack, I can approve this with its current structure (same playbook for both versions) but I strongly advise to change it to use 2 playbooks as I suggested above (even after this PR is merged in a different PR as an enhancement) since this pack is used by a lot of customers and published by XSOAR's CS team.
Also, I assume you did this already, but I strongly recommend to verify the playbook works on both versions.

> - Update main playbook to differentiate between x6 and x8 versions.
> 
> #### Scripts
> 
> ##### HealthCheckAPIvalidation
> 
> - Updated the docker image to **demisto/python3:3.11.10.111526**.
> 
> ##### HealthCheckAnalyzeLargeInvestigations
> 
> - Updated the docker image to **demisto/python3:3.11.10.111526**.

Reviewer 2: ```suggestion
Updated the main playbook to differentiate between Cortex XSOAR versions 6.x and 8.x.

#### Scripts

##### HealthCheckAPIvalidation

Updated the Docker image to **demisto/python3:3.11.10.111526**.

##### HealthCheckAnalyzeLargeInvestigations

Updated the Docker image to **demisto/python3:3.11.10.111526**.
```

##### HealthCheckCPU

> - Implemented notification for x8 incompatibility.

Reviewer 2: ```suggestion
- Implemented notification for Cortex XSOAR version 8.x incompatibility.
```

> - Updated the docker image to **demisto/python3:3.11.10.111526**

Reviewer 2: ```suggestion
- Updated the Docker image to **demisto/python3:3.11.10.111526**
```

##### HealthCheckCommonIndicators

> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckContainersStatus
> 
> - Implemented notification for x8 incompatibility.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckDiskUsage
> 
> - Implemented notification for x8 incompatibility.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckDiskUsageLine
> 
> - Implemented notification for x8 incompatibility.
> - Updated the docker image to **demisto/python3:3.11.10.111526**

Reviewer 2: ```suggestion
Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckContainersStatus

- Implemented notification for Cortex XSOAR version 8.x incompatibility.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckDiskUsage

- Implemented notification for Cortex XSOAR version 8.x incompatibility.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckDiskUsageLine

- Implemented notification for Cortex XSOAR version 8.x incompatibility.
- Updated the Docker image to **demisto/python3:3.11.10.111526**
```

##### HealthCheckDockerLog

> - Implemented notification for x8 incompatibility.
> 
> ##### HealthCheckFields
> 
> - Fixing typo.
> 
> ##### HealthCheckFileSystem
> 
> - Implemented notification for x8 incompatibility.

Reviewer 2: ```suggestion
Implemented notification for Cortex XSOAR version 8.x incompatibility.

##### HealthCheckFields

Fixed typo.

##### HealthCheckFileSystem

Implemented notification for Cortex XSOAR version 8.x incompatibility.
```

##### HealthCheckGetLargestInputsAndOutputsInIncidents

> - Fixing script for x6

Reviewer 2: ```suggestion
- Fixed script for Cortex XSOAR version 6.x.
```

- Implemented notification for x8 incompatibility.

##### HealthCheckIncidentTypes

> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckIncidentsCreatedDaily
> 
> - Added support for x8 version.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckIncidentsCreatedMonthly
> 
> - Added support for x8 version.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckIncidentsCreatedWeekly
> 
> - Added support for x8 version.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckInstalledPacks
> 
> - Added support for x8 version.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckIntegrations
> 
> - Updated the docker image to **demisto/python3:3.11.10.111526**

Reviewer 2: ```suggestion
Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckIncidentsCreatedDaily

- Added support for Cortex XSOAR version 8.x.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckIncidentsCreatedMonthly

- Added support for Cortex XSOAR version 8.x.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckIncidentsCreatedWeekly

- Added support for Cortex XSOAR version 8.x.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckInstalledPacks

- Added support for Cortex XSOAR version 8.x.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckIntegrations

Updated the Docker image to **demisto/python3:3.11.10.111526**
```

##### HealthCheckLicenseData

> - Implemented notification for x8 incompatibility.
> 
> ##### HealthCheckMemory
> 
> - Implemented notification for x8 incompatibility.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckNumberOfDroppedIncidents
> 
> - Added support for x8 version.
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckNumberOfLargeIOIncidents
> 
> - Implemented notification for x8 incompatibility.
> 
> ##### HealthCheckNumberOfLargeIncidents
> 
> - Implemented notification for x8 incompatibility.
> 
> ##### HealthCheckNumberOfPartitions
> 
> - Implemented notification for x8 incompatibility.

Reviewer 2: ```suggestion
Implemented notification for Cortex XSOAR version 8.x incompatibility.

##### HealthCheckMemory

- Implemented notification for Cortex XSOAR version 8.x incompatibility.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckNumberOfDroppedIncidents

- Added support for Cortex XSOAR version 8.x.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckNumberOfLargeIOIncidents

Implemented notification for Cortex XSOAR version 8.x incompatibility.

##### HealthCheckNumberOfLargeIncidents

Implemented notification for Cortex XSOAR version 8.x incompatibility.

##### HealthCheckNumberOfPartitions

Implemented notification for Cortex XSOAR version 8.x incompatibility.
```

##### HealthCheckPlaybookAnalysis

> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckServerConfiguration
> 
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckSystemDiagnostics
> 
> - Revised and updated code
> - Updated the docker image to **demisto/python3:3.11.10.111526**
> 
> ##### HealthCheckWorkers
> 
> - Revised and updated code
> - Updated the docker image to **demisto/python3:3.11.10.111526**

Reviewer 2: ```suggestion
Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckServerConfiguration

Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckSystemDiagnostics

- Revised and updated the code.
- Updated the Docker image to **demisto/python3:3.11.10.111526**

##### HealthCheckWorkers

- Revised and updated the code.
- Updated the Docker image to **demisto/python3:3.11.10.111526**
```

> 

Reviewer 3: I made the RN format, please replace each - %%UPDATE_RN%% with the related release note.

#### Playbooks

##### HealthCheck

> - %%UPDATE_RN%%

Reviewer: @tsachzim Please add the changes you made for this and everywhere in this readme where it says "%%UPDATE_RN%%"
